.changeset/smart-turkeys-bathe.md

```diff
@@ -0,0 +1,30 @@
+---
+'@backstage/plugin-catalog-backend': minor
+'@backstage/plugin-catalog-import': minor
```

Comment:
missing backend-common here which is the big one that did get the change

Reply:
I didn't see any files changed in the packages/backend-common directory, I've selected only plugins/packages which got files changed.

Reply:
Ah yes you are quite right, sorry i completely mixed it up, of course the builder is in plugin-catalog-backend.